noxfile.py

```diff
@@ -259,7 +259,7 @@ def cover(session):
     test runs (not system test runs), and then erases coverage data.
     """
     session.install("coverage", "pytest-cov")
-    session.run("coverage", "report", "--show-missing", "--fail-under=88")
+    session.run("coverage", "report", "--show-missing", "--fail-under=89")
```

Comment:
I hope this gets bumped to 100% soon, as anything less is susceptible to false positives. For example, refactoring (shortening) a piece of tested code would cause a drop in the overall coverage.

Reply:
https://github.com/googleapis/python-bigquery-pandas/issues/392

I plan to tackle this next, as I do hope the next pandas-gbq release can be a 1.0 release, but wouldn't be comfortable without full test coverage.